packages/atomic/src/locales.json

```diff
@@ -483,6 +483,14 @@
     "zh-cn": "没有找到与 {{query}} 相关的结果",
     "zh-tw": "沒有找到與 {{query}} 相關的結果"
   },
+  "no-results-for-v1": {
```

Comment:
I think modifying the original values for this is fine 🤷  I don't see modifying strings as something breaking implementation / causing errors, etc.

Reply:
Am I also responsible for doing all the translations?

Reply:
No, just English/French is perfect 👍